src/StoreKeeper/WooCommerce/B2C/Migrations/Versions/V20230319124000PaymentsAddIsPaid.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Migrations\Versions;
+
+use StoreKeeper\WooCommerce\B2C\Database\DatabaseConnection;
+use StoreKeeper\WooCommerce\B2C\Models\PaymentModel;
+
+class V20230319124000PaymentsAddIsPaid extends \StoreKeeper\WooCommerce\B2C\Migrations\AbstractMigration
```

Comment:
Can we change all instances of  qualifier `extends \StoreKeeper\WooCommerce\B2C\Migrations\AbstractMigration` to import `extends AbstractMigration` just for the sake of my OCD? :sweat_smile:

Reply:
done